pycm/pycm_param.py

```diff
@@ -18,6 +18,36 @@
 
 '''
 
+OG_IMAGE_URL = "http://www.pycm.ir/images/logo-og.png"
+
+OG_DESCRIPTION = "PyCM is a multi-class confusion matrix library written in Python. http://www.pycm.ir"
+
+HTML_INIT_TEMPLATE = '''
+<!doctype html>
+<html lang="en">
+<head>
```

Comment:
There were a `\n` after `<head>` in latest `dev` version, is it removed on purpose?

Reply:
No need to extra `\n` in a multi-line string

Reply:
fixed in https://github.com/sepandhaghighi/pycm/pull/368/commits/868923e82cc48767fb62fb9fcb61823fe031a432